msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -112,6 +114,110 @@ Test Tool offers a faster debug experience for bot applications when compared to
 
    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="../assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool.":::
 
+# [Command line](#tab/cli)
+1. To debug your Teams app test tool, use command line in JavaScript, TypeScript, or C# as follows:
+   # [JavaScript/TypeScript](#tab/clijs)
+
+   1. Run the following command to install Teams App Test Tool CLI from `npm`:
+
+      ```cmd
+      npm install -g @microsoft/teams-app-test-tool
+      ```
+
+        :::image type="content" source="../assets/images/teams-toolkit-v2/debug/npm-teams-app-test-tool.png" alt-text="Screenshot shows install teams app test tool cli.":::
+
+   1. Use the `teamsapp` CLI to create your first project. Start from the folder where you want to create the project folder.
```

Comment:
Better to clarify the `teamsapp` CLI is from TeamsFx CLI instead of Teams App Test Tool CLI, and user requires to install TeamsFx CLI before using the following command.